policyengine_us/variables/gov/states/co/tax/income/credits/sales_tax_refund/co_sales_tax_refund_eligible.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class co_sales_tax_refund_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Colorado sales tax refund"
+    definition_period = YEAR
+    reference = "https://tax.colorado.gov/sites/tax/files/documents/DR_0104_Book_2022.pdf#page=23"
+    defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.co.tax.income.credits.sales_tax_refund
+        age_head = tax_unit("age_head", period)
+        head_eligible = age_head >= p.age_threshold
+        age_spouse = tax_unit("age_spouse", period)
+        spouse_eligible = age_spouse >= p.age_threshold
+        income_tax = tax_unit(
```

Comment:
I'd suggest also checking if employment_income > 0, as a proxy for getting a refund for withheld taxes, the other pathway to eligibility. And file an issue to capture this more accurately.
```suggestion
        # Legal code is ambiguous, but the form points to the line corresponding to
        # tax before non-refundable credits.
        income_tax = tax_unit(
```